src/store/admin/types.ts

```diff
@@ -84,6 +84,7 @@ export interface GlobalMute {
 
 export interface GloballyMutedPlayer extends GlobalMute {
   id: string;
+  createdAt: string;
```

Comment:
How come this is part of GloballyMutedPlayer and not GlobalMute?

Reply:
Because it's not known at the time of creation